### PR TITLE
Updating linux HW requirements

### PR DESCRIPTION
**Pull request by majo-javorka**
- changed Ubuntu distributions to 18 and 20
- added supported desktop environments and display server

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/hardware-requirements-for-the-teams-app.md [hunk 1]
```diff
@@ -69,7 +69,9 @@ All of the requirements in the following sections apply to both the Microsoft Te
 |Devices    |   Standard laptop camera, microphone, and speakers    |
 |Voice    |  Compatible microphone and speakers, headset with microphone, or equivalent device       |
 |Video calls and meetings | <ul><li>Requires 2-core processor. For higher video/screen share resolution and frame rate, a 4-core processor or better is recommended.</li><li>Joining a meeting using proximity detection in a Microsoft Teams Room is not available on Linux.</li></ul>
-|Supported Linux distributions | Ubuntu 16.04 LTS, 18.04 LTS, Fedora 30 Workstation, RHEL 8 Workstation, CentOS 8
+|Supported Linux distributions | Ubuntu 18.04 LTS, 20.04 LTS, Fedora 30 Workstation, RHEL 8 Workstation, CentOS 8       |
+|Supported Desktop environment | GNOME, KDE       |
```
Comment (JohanFreelancer9) [inline Teams/hardware-requirements-for-the-teams-app.md:73]:
```suggestion
|Supported desktop environment | GNOME, KDE       |
```

File: Teams/hardware-requirements-for-the-teams-app.md [hunk 2]
```diff
@@ -69,7 +69,9 @@ All of the requirements in the following sections apply to both the Microsoft Te
...
+|Supported Display server | X11       |
```
Comment (JohanFreelancer9) [inline Teams/hardware-requirements-for-the-teams-app.md:74]:
```suggestion
|Supported display server | X11       |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**